flang/runtime/stop.cpp

```diff
@@ -152,11 +156,39 @@ void RTNAME(PauseStatementText)(const char *code, std::size_t length) {
   std::exit(status);
 }
 
+static void PrintBacktrace() {
+#ifdef __linux__
+  // TODO: Need to parse DWARF information to print function line numbers
+  constexpr int MAX_CALL_STACK{999};
+  void *buffer[MAX_CALL_STACK];
+  int nptrs{backtrace(buffer, MAX_CALL_STACK)};
+  char **symbols{backtrace_symbols(buffer, nptrs)};
+
+  if (symbols == nullptr) {
+    Fortran::runtime::Terminator{}.Crash("no symbols");
+    std::exit(EXIT_FAILURE);
```

Comment:
We shouldn't exit here, just not print any symbols and return from the function.  You can just do
```
if (char **symbols{...}) {
  for (...) {
    print symbol[i]
  }
  free(symbols);
}
```